app/views/likes/destroy.js.erb

```diff
@@ -0,0 +1,13 @@
+console.log("Destroyed!")
+
+var existingLikes = $("#<%= dom_id(@like.photo) %>_likes");
+
+var updatedLikes = $("<%= j(render "photos/likes", photo: @like.photo) %>");
+
+existingLikes.replaceWith(updatedLikes)
+
+var icon = $("#<%= dom_id(@like.photo) %>_likes_icon")
```

Comment:
Introducing situational variables inside a JS file is a nice way to build the response logic for an AJAX request. I didn't think this way, thanks for showing new ways of doing things!